### PR TITLE
Added docker build code

### PR DESCRIPTION
**Pull request by Sandhya1874**

### CODE REVIEW

File: .github/workflows/test_docker_build.yml
Comment (james-jdgtl) [inline .github/workflows/test_docker_build.yml:1]: Really dumb question, but is there any reason why you've prefixed the workflow names with test_ ..?

To keep them grouped in the - doubtless growing collection of workflows, may I recommend using
`build_docker.yml` for this one and
`build_multiplatform_docker` for the other one?

**Review by james-jdgtl — CHANGES_REQUESTED** [review]
Really dumb question, but is there any reason why you've prefixed the workflow names with test_ ..?

To keep them grouped in the - doubtless growing collection of workflows, may I recommend using
build_docker.yml for this one and
build_multiplatform_docker for the other one?

File: .github/workflows/test_multiplatform_docker_build.yml
Comment (james-jdgtl) [inline .github/workflows/test_multiplatform_docker_build.yml:1]: recommending `build_multiplatform_docker.yml` for this one

File: .github/workflows/test_docker_build.yml
Comment (Sandhya1874) [reply .github/workflows/test_docker_build.yml:1]: Renamed file name

**Review by james-jdgtl — APPROVED** [review]

**Pull request merged**